### DISCUSSION TITLE
How can I convert TIME to INTEGER?

### DISCUSSION THREAD

**Discussion by ved-genmo** [duckdb/duckdb#11389] [Q&A]
I have values that are of datatype TIME. How can I convert them to integer (I want to calculate the difference between 2 TIME values)?

**Comment by soerenwolfers** [comment]
Did you consult
https://duckdb.org/docs/sql/functions/time.html
https://duckdb.org/docs/sql/functions/timestamp.html
https://duckdb.org/docs/sql/functions/interval.html

**Reply by derhuerst** [reply]
I want to compute the square of the seconds between two `TIMESTAMPTZ`s. How do I do this?
